db/migrations/004_incidents.sql

```diff
@@ -89,16 +89,24 @@ CREATE TABLE comments (
   id UUID DEFAULT generate_ulid() PRIMARY KEY,
   created_by UUID NOT NULL ,
   comment text NOT NULL,
-  external_id TEXT NULL, -- A unique identifier for the responder in the external system e.g. Jira ticket id
   incident_id UUID NOT NULL,
-  responder_id UUID NULL,
```

Comment:
I think we still need a responder_id, this will be the source of the comment

Reply:
Oh, for comments made from responder and added here ? I guess so, in that case we should put back the external_id as well here.